### PR TITLE
Add geolocation data

### PR DESCRIPTION
**Pull request by makew0rld**
For https://github.com/starlinglab/integrity-backend/issues/120

- ~~[ ] Proofmode~~ removed because geolocation info is not added, see [here](https://github.com/starlinglab/integrity-schema/pull/16)
- [x] Testing

### CODE REVIEW

File: http/main.py
Comment (benhylau) [inline http/main.py:210]: 1. Based on current implementation if `geocoder.osm([lat, lon], method="reverse").status_code != 200` it'd throw, but we are not catching here. The whole block will fail.
2. What if `reverse_geocode` returns None? Is that possible?

**Review by benhylau — CHANGES_REQUESTED** [review]
Please check exception and general err handling before merge.

File: http/main.py
Comment (makew0rld) [reply http/main.py:210]: 1. Yes, the catching is done in the `create` function, so the whole request would fail and a 500 error would be returned the client. Should the request proceed without reverse-geolocation instead?
2. That is possible, but will raise a `TypeError`, so the same behaviour described above will occur.

Comment (benhylau) [reply http/main.py:210]: Discussed this in call. This is a user facing error and we want it to fail. @makew0rld will investigate if there are legitimate cases (e.g. the info about the lat/lon is actually not available) that leads to a thrown exception. We need to handle the legitimate cases, but ok to throw when e.g. service outage.

Comment (makew0rld) [reply http/main.py:210]: Skip reverse-geocoding if lat/lon doesn't have a valid address. Merge this after Yurko reviews.

**Comment by YurkoWasHere** [comment]
Test failing, will fix it Dev.

**Pull request merged**
